docs/src/DatePicker.doc.js

```diff
@@ -0,0 +1,399 @@
+    description="
+    You can use this component for a visual divider between two elements.
+  "
+    name="Example: Date range picker"
+    defaultCode={`
+function DatePickerRangeExample() {
+  const [startDate, setStartDate] = React.useState(undefined);
+  const [endDate, setEndDate] = React.useState(undefined);
+
+  const handleSetStartDate = ({ event, value }) => setStartDate(value);
+  const handleSetEndDate = ({ event, value }) => setEndDate(value);
+
+  return (
+    <Box>
+      <Box
+        display="flex"
+        justifyContent="between"
+        marginStart={-1}
+        marginEnd={-1}
+      >
+        <Box column={6} paddingX={1}>
+          <DatePicker
+            startDate={startDate}
+            accessibilityLabelNext='Go to next month'
+            accessibilityLabelPrevious='Go to previous month'
+            accessibilityLabelIcon='Select a start date'
+            endDate={endDate}
+            id='1'
+            idealDirection="up"
+            onChange={handleSetStartDate}
+
+            placeholder='Start Date'
+            selectsStart
+            value={startDate}
+          />
+        </Box>
+        <Box column={6} paddingX={1}>
+          <DatePicker
+            startDate={startDate}
+            accessibilityLabelNext='Go to next month'
+            accessibilityLabelPrevious='Go to previous month'
+            accessibilityLabelIcon='Select an end date'
+            endDate={endDate}
+            id='2'
+            idealDirection="up"
+            onChange={handleSetEndDate}
+
+            placeholder='End Date'
+            selectsEnd
+            value={endDate}
+          />
+        </Box>
+      </Box>
+    </Box>
+  );
+}
+`}
+  />
+);
+
+card(
+  <Example
+    id="example-disabled"
+    description="
+    DatePicker can display its own helper message for cases where you want to provide more information about the date field.
+  "
+    name="Example: Disabled"
+    defaultCode={`
+function DatePickerExample() {
+  const [date, setDate] = React.useState(new Date());
+  return (
+    <DatePicker
+      accessibilityLabelNext='fakeAccessibilityLabelNext'
+      accessibilityLabelPrevious='fakeAccessibilityLabelPrevious'
+      accessibilityLabelIcon='fakeAccessibilityLabelIcon'
+      disabled
+      id="disabled"
+      onChange={({value}) => setDate(value)}
+      value={date}
+    />
+
+  )
+}
+`}
+  />
+);
+
+card(
+  <Example
+    id="helperText"
+    description="
+    DatePicker can display its own helper message for cases where you want to provide more information about the date field.
+  "
+    name="Example: helperText"
+    defaultCode={`
+function DatePickerExample() {
+  const [date, setDate] = React.useState(new Date());
+  return (
+    <DatePicker
+      accessibilityLabelNext='fakeAccessibilityLabelNext'
+      accessibilityLabelPrevious='fakeAccessibilityLabelPrevious'
+      accessibilityLabelIcon='fakeAccessibilityLabelIcon'
+      helperText={'https://pinterest.com/'}
```

Comment:
@AlbertCarreras could we provide a better helperText? E.g. something like `Use mm/dd/yyyy` as the date format

Reply:
Fixed